src/hotspot/cpu/x86/macroAssembler_x86_exp.cpp

```diff
@@ -493,7 +493,7 @@ void MacroAssembler::fast_exp(XMMRegister xmm0, XMMRegister xmm1, XMMRegister xm
   subl(rsp, 120);
   movl(Address(rsp, 64), tmp);
   lea(tmp, ExternalAddress(static_const_table));
-  movdqu(xmm0, Address(rsp, 128));
+  movsd(xmm0, Address(rsp, 128));
```

Comment:
Can you explain this change?

Reply:
Would be nice to add comment about what values are on stack.

Reply:
movdqu moves 128 bits from the memory, while movsd moves 64 bits. movsd is what's needed for double precision calculation.
In this case however, no harm was done even using movdqu, as the subsequent vunpcklpd would broadcast only the lower 64bits. Still it is safe to change to movsd to begin with

an example of stack is c05ec00000000000. movqqu would move 0x00000000e719ee40c05ec00000000000